tooling/operation.js

```diff
@@ -8,11 +8,12 @@ const getResponseExamples = require('./operation/get-response-examples');
 const matchesMimeType = require('./lib/matches-mimetype');
 
 class Operation {
-  constructor(oas, path, method, operation) {
+  constructor(oas, path, method, operation, jwtDefaults) {
```

Comment:
It feels weird tying our JWT naming into this. Maybe this parameter and `this.jwtDefaults` should be called `globalDefaults` instead? Or instead of attaching it to the `Operation` class, it should just be passed into `parametersAsJsonSchema(jwtDefaults)`?

Reply:
I just extracted it out of the Operation class and decided to pass it through `getParametersAsJsonSchema()`!